civis/io/_tables.py

```diff
@@ -578,7 +578,8 @@ def civis_to_multifile_csv(sql, database, job_name=None, api_key=None,
     >>> manifest = civis_to_multifile_csv(sql, database, delimiter=delimiter)
     >>> ids = [entry['id'] for entry in manifest['entries']]
     >>> buf = BytesIO()
-    >>> civis_to_file(ids[0], buf)
+    >>> for file_id in ids:
+    >>>     civis_to_file(file_id, buf)  # write each file to buffer
     >>> buf.seek(0)
     >>> df = pd.read_csv(buf, delimiter=delimiter)
```

Comment:
Sorry, I read the PR too fast previously. The purpose of `civis_to_multifile_csv` is to unload a large amount of data so that we can work with it in chunks after the unloading is done. So the `for` loop you're adding would need a new `BytesIO()` instantiation within each iteration, rather than shovel all the data into a single `buf` object (which may blow up memory, defeating the whole point of `civis_to_multifile_csv`). What do you think about something like this?

```python
for file_id in ids:
    buf = BytesIO()
    civis_to_file(file_id, buf)
    buf.seek(0)
    # Process the data in `buf` for your own application, e.g., using pandas...
    df = pd.read_csv(buf, delimiter=delimiter)
```